api/rest.go

```diff
@@ -3436,14 +3444,15 @@ func handleCreateSource(exec boil.Executor, r CreateSourceRequest) (*Source, *Ht
 			return nil, NewInternalError(err)
 		}
 	}
+	common.CreateCUTypeSource(&s.Source, exec)
 
 	return handleGetSource(exec, s.ID)
 }
 
 func handleGetSource(exec boil.Executor, id int64) (*Source, *HttpError) {
 	source, err := models.Sources(exec,
 		qm.Where("id = ?", id),
-		qm.Load("SourceI18ns")).
+		qm.Load("SourceI18ns", "ContentUnits")).
```

Comment:
Very wasteful db access. No need to load all units every time we need a source

Reply:
fixed